tests/test_wps_parameters.py

```diff
@@ -21,3 +21,19 @@ def test_parameters_local(config):
         temp_config.read()
         params = f"config={temp_config.name};"
         run_wps_process(Parameters(), params)
+
+
+@pytest.mark.online
+@pytest.mark.parametrize(
+    ("config"), [resource_filename(__name__, "configs/parameter_https.cfg")],
+)
+def test_parameters_https(config, make_mock_urls):
```

Comment:
What does each `FILE_PATH` in your `parameter_https.cfg` file look like? I just realized that running the notebook as is permanently changes it.

Reply:
Actually, the demo doesn't use that `.cfg` file, so it won't be modified. I'll look into it more.

Reply:
Could you try [replacing self.workdir in this line with a different directory](https://github.com/pacificclimate/osprey/blob/4283862881ddc828fa4f43acdb47342ce73a6ae7/osprey/processes/wps_parameters.py#L101), and [after the for loop in replace_urls, print filedata](https://github.com/pacificclimate/osprey/blob/4283862881ddc828fa4f43acdb47342ce73a6ae7/osprey/utils.py#L64)? I just want to see if the contents of the config file are being modified correctly.

Reply:
> What does each FILE_PATH in your parameter_https.cfg file look like? I just realized that running the notebook as is permanently changes it.

I think you are right here. I did `git stash` and ran pytest again, and it passed. Last time I ran the demo first, so that caused the issue you said.